litellm/proxy/_types.py

```diff
@@ -1818,7 +1818,7 @@ def __init__(
         code: Optional[Union[int, str]] = None,
         headers: Optional[Dict[str, str]] = None,
     ):
-        self.message = message
+        self.message = message.decode('utf-8') if isinstance(message, bytes) else message
```

Comment:
can you add this logic below, let's leave this line to just be `self.message = message`